frappe/deferred_insert.py

```diff
@@ -5,7 +5,6 @@
 
 queue_prefix = 'insert_queue_for_'
 
-@frappe.whitelist()
```

Comment:
Hope this doesn't break other apps :trollface: 

---

Also, how about moving `save_to_db` from an hourly job to say every 5-15 mins?

Reply:
> Also, how about moving save_to_db from an hourly job to say every 5-15 mins?

Not touching it right now... can be part of another PR.